src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractReportServiceImpl.java

```diff
@@ -220,13 +221,13 @@ protected String buildHeaderBreakFromHeader(Object header) {
         return StringUtils.repeat(KFSConstants.DASH, header.toString().length());
     }
 
-    protected void writeErrorSubReportWithNoLineItems(String subTitle, String message) {
-        writeSubTitleForErrorSubReport(subTitle);
+    protected void writeErrorSubReportWithNoLineItems(String[] subTitles, String message) {
```

Comment:
I think it would be cleaner to make "subTitles" a var-args parameter, so that you don't have to explicitly construct the array when calling this method. However, this would require reordering the parameters so that the var-args one is last, so care would be needed to update the calling code accordingly.

Reply:
good idea, I shall re work this